### DISCUSSION TITLE
Multi-select *vertical* scroll?

### DISCUSSION THREAD

**Discussion by michael-mcgreal** [onesine/react-tailwindcss-datepicker#22] [Q&A] [answered]
Is it possible to display a dateRange picker where the **months are stacked vertically**? 

Similar to how HTML's default [date input](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date) scrolls vertically on month change?

https://user-images.githubusercontent.com/11381911/209709203-5ac3bf85-4839-411f-b4a2-81fe35837ca8.mp4

Thank you!

**Comment by onesine** [comment, marked as answer]
Hello @mcgrealife 👋, thank you for your feedback.

We will include your request in the list of features to do. It will arrive very soon in the new updates.

**Reply by michael-mcgreal** [reply]
Wow that is superb – thanks @onesine and team!
